dom/src/main/scala/org/scalajs/dom/DataTransferItem.scala

```diff
@@ -0,0 +1,26 @@
+/** All documentation for facades is thanks to Mozilla Contributors at https://developer.mozilla.org/en-US/docs/Web/API
+  * and available under the Creative Commons Attribution-ShareAlike v2.5 or later.
+  * http://creativecommons.org/licenses/by-sa/2.5/
+  *
+  * Everything else is under the MIT License http://opensource.org/licenses/MIT
+  */
+package org.scalajs.dom
+
+import scala.scalajs.js
+
+/** Each DataTransferItem object is associated with a [[DataTransfer]] object. */
+@js.native
+trait DataTransferItem extends js.Object {
```

Comment:
It appears to be a `class` with a non-public constructor.

```suggestion
class DataTransferItem private[this] extends js.Object {
```